_data/toc.yaml

```diff
@@ -1842,16 +1842,12 @@ manuals:
         title: Manage
   - path: /docker-hub/official_images/
     title: Docker Official images
-  - sectiontitle: Docker Verified Publisher
-    section:
-      - path: /docker-hub/publish/
-        title: Overview
-      - path: /docker-hub/publish/insights-analytics/
-        title: Insights and analytics
-      - path: /docker-hub/publish/repository-logos/
```

Comment:
Is this an old page that should get deleted as well?

Reply:
@docker/docs-team any thoughts on if there's a better way to do this? Or just remove the page? As I don't think it has received much traffic anyway.

Reply:
I would just remove the page. It has a lot of inaccurate info on it and now that it has been pulled out of the sub-folder it'll be more visible which is not what we want